.github/workflows/heroku.yml

```
@@ -32,6 +32,7 @@ jobs:
       run: |
         deploymentId=$(jq -n "{
             \"ref\": \"${{ inputs.branch }}\",\
+            \"auto_merge\": \"false\",\
```

Because this is JSON? I haven't tested this or checked if GitHub API makes a difference

```suggestion
            \"auto_merge\": false,\
```

---

Ah, of course, it's not a string 🤦‍♂️.

Fixed https://github.com/84codes/actions/commit/79ad8e4e78efe663b0f8b190bc4fabbfd1a68d91